### DISCUSSION TITLE
deepmd-kit package is inconsitent with TensorFlow Runtime

### DISCUSSION THREAD

**Discussion by WU2JING** [deepmodeling/deepmd-kit#1869] [General]
I successfully install deepmd-kit 2.1.4, but the error  raised as follow, when train a model.
![image](https://user-images.githubusercontent.com/68996784/185735755-3a19b5c7-e050-461a-9b73-3deaa2f3ab4e.png)

**Comment by wanghan-iapcm** [comment]
@njzjz could you please take a look?

**Comment by njzjz** [comment]
How did you install deepmd-kit? Did you use devtoolset on RHEL6/7 as is shown in the error message?

**Reply by WU2JING** [reply]
I install deepmd-kit with conda command and set CONDA_OVERRIDE_CUDA="11.6"

**Reply by njzjz** [reply]
Failed to reproduce: https://nbviewer.org/gist/njzjz/e660ea2d5469d6b1a2fb44e306bcbc38

**Reply by njzjz** [reply]
You may check your packages (`conda list`, `pip list`) and environment variables (`env`) to see if anything is wrong.

**Reply by WU2JING** [reply]
Ok, thank you.

**Reply by WU2JING** [reply]
It works, thanks again.

**Comment by samuelzyzhang** [comment]
> I successfully install deepmd-kit 2.1.4, but the error raised as follow, when train a model. ![image](https://user-images.githubusercontent.com/68996784/185735755-3a19b5c7-e050-461a-9b73-3deaa2f3ab4e.png)

I encountered the same problem as yours, how did you solve it?

**Comment by WU2JING** [comment]
You can try to use the fllow command.
CONDA_OVERRIDE_CUDA=11.6 `pwd`/miniconda3/bin/conda create -n deepmd deepmd-kit=*=*gpu -c https://conda.deepmodeling.org -y
